openshift/topics/tutorials.adoc

```diff
@@ -1,30 +1,36 @@
 == Tutorials
 
+[role="_abstract"]
+The tutorials in this chapter assume that you have an OpenShift instance similar to the one created by performing link:{ocpdocs_install_cluster_link}[the installation of the OpenShift Container Platform cluster].
+
 [[upgrading-sso-db-from-previous-version]]
-=== Example Workflow: Updating Existing Database to Migrate to New {project_openshift_product_name} Image Version
-[IMPORTANT]
-====
-* Rolling updates from previous versions of {project_openshift_product_name} to version {project_version} are not supported as databases and caches are not backward compatible.
-* Stop all instances running some of previous versions of the {project_openshift_product_name} before upgrading. They cannot run concurrently against the same database.
-* Pre-generated scripts are not available, they are generated dynamically depending on the database.
-====
+=== Updating a database for a new {project_openshift_product_name} image version
+
+Note the following points related to the update:
+
+* Rolling updates from a previous versions of {project_openshift_product_name} to version {project_version} are not supported as databases and caches are not backward compatible.
+* Instances from versions of the {project_openshift_product_name} cannot be runnng before upgrade. They cannot run concurrently against the same database.
+* Pre-generated scripts are not available. They are generated dynamically depending on the database.
 
-{project_name} {project_version} can xref:automatic-db-migration[automatically migrate the database schema], or you can choose to do it xref:manual-db-migration[manually].
+You have two choices for updating the database:
+
+* Allow {project_name} {project_version} to xref:automatic-db-migration[automatically migrate the database schema]
+* Update the database xref:manual-db-migration[manually]
```

Comment:
Should this contain `or`, like `*Or, update the database xref:manual-db-migration[manually]`? (is it clear enough from the form above, those are alternative options?)

Reply:
I think it's clear.  It is either "automatic" or "manual."